packages/video_player/video_player/lib/video_player.dart

```diff
@@ -784,7 +784,8 @@ class _VideoScrubberState extends State<_VideoScrubber> {
         seekToRelativePosition(details.globalPosition);
       },
       onHorizontalDragEnd: (DragEndDetails details) {
-        if (_controllerWasPlaying) {
+        if (_controllerWasPlaying &&
+            controller.value.position != controller.value.duration) {
```

Comment:
I was looking at how this is handled in QT Player and it seems they also use the "looping" setting to determine what to do:

* `wasPlaying + scrolledToEnd + looping` : jump and pause in FIRST frame of the video.
* `wasPlaying + scrolledToEnd + !looping` : pause in last frame. This is the same as `!wasPlaying + scrolledToEnd`, regardless of value of looping.)

In all cases above, clicking "Play", immediately starts the video from the beginning.

In no case, QT restarted when scrubbing to the end.